block-switch/block-switch-core/src/main/resources/blockSwitch.js

```diff
@@ -46,6 +47,7 @@ function globalSwitch() {
 	$('.switch--item').each(function() {
 		$(this).off('click');
 		$(this).on('click', function() {
+			window.localStorage.setItem("blockSwitch", $(this).text());
```

Comment:
Thanks for the PR, Jay. I'm not sure that this will work for documentation with more than one set of switches. For example, [REST Docs](https://docs.spring.io/spring-restdocs/docs/2.0.4.RELEASE/reference/html5/) has two sets:

- Gradle and Maven
- MockMvc, REST Assured, and WebTestClient

If you switched to `Gradle` in one part of the documentation and then switched to `MockMvc` in another, won't your selection of `Gradle` then be forgotten?

I think we need to iterate over all of the switches and group them based on the choices that they offer. A value then needs to be stored for each distinct set of choices.

Reply:
OK. I'll look into that. Thanks.